packages/react-client/src/forks/ReactFlightClientHostConfig.dom-node.js

```diff
@@ -9,4 +9,4 @@
 
 export * from 'react-client/src/ReactFlightClientHostConfigBrowser';
 export * from 'react-client/src/ReactFlightClientHostConfigStream';
-export * from 'react-server-dom-webpack/src/ReactFlightClientWebpackBundlerConfig';
+export * from 'react-server-dom-webpack/src/ReactFlightClientWebpackBundlerConfig.js';
```

Comment:
I added the '.js' file extension to the import path to fix the ENOENT error. But it is not clear why the '.js' file extension is not added automatically

<img width="1093" alt="image" src="https://user-images.githubusercontent.com/22427967/218251753-bd10a75b-eda0-490a-9738-0bb61735b197.png">

Reply:
These imports seem fragile to me and easy to mess. Why do we now need in some places the explicit `.js` suffix?

Does it have maybe something to do with this line below?

> moduleSideEffects: id => !moduleSideEffects.includes(id),

where the `id` only matches if the `.js` exists? (Just a guess, I don't really understand the config 😄 )